apps/event_processors/models.py

```
@@ -1 +1,139 @@
+    )
+    channel_id = models.CharField(
+        max_length=50,
+        help_text="PUI of source channel (e.g., 'NODE3000004')",
+    )
+    detected_at = models.DateTimeField(
+        help_text="Logical timestamp of event occurrence",
+    )
+    confidence = models.DecimalField(
+        max_digits=3,
+        decimal_places=2,
+        help_text="Detection confidence (0.00–1.00)",
```

Ruff (0.14.14)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)